packages/contracts-bedrock/scripts/libraries/L2GenesisHelpers.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { Predeploys } from "src/libraries/Predeploys.sol";
+
+/// @title L2GenesisHelpers
+/// @notice Contains constants and helper methods that are commonly used for generating and testing L2 genesis.
+library L2GenesisHelpers {
+    uint256 constant PRECOMPILE_COUNT = 256;
+    uint256 constant PROXY_COUNT = 2048;
+
+    /// @notice The storage slot that holds the address of the owner.
+    /// @dev `bytes32(uint256(keccak256('eip1967.proxy.admin')) - 1)`
+    bytes32 internal constant PROXY_ADMIN_ADDRESS = 0xb53127684a568b3173ae13b9f8a6016e243e63b6e8ee1178d6a717850b5d6103;
```

Comment:
the 1967 constants are defined many times in the codebase, i believe there are at least 3 diff constants that have them